app/pages/GoodsReferencePage.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import models.Index
+import models.reference.Item
+import pages.sections.TransportEquipmentSection
+import play.api.libs.json.JsPath
+
+final case class GoodsReferencePage(equipmentIndex: Index) extends QuestionPage[Item] {
+
+  override def path: JsPath = TransportEquipmentSection(equipmentIndex).path \ "GoodsReference" \ toString
+
+  override def toString: String = "consignmentItem"
```

Comment:
Should it be this one: `declarationGoodsItemNumber`, maybe I looked in the wrong place

Reply:
I think it should probably be "goodsReference" actually, it's the parent level with both "declarationGoodsItemNumber" and "sequenceNumber" within it. Updated